watchdog/tests/get_config.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
I think you want to add these tests to the github workflow file. Otherwise, they won't run in CI.

Reply:
Done, thank you!